source/form.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html class="form" lang="ru">
+<head>
+  <meta charset="UTF-8">
+  <title>HTML Academy: Кэт энерджи</title>
+</head>
+<body class="form__body">
+  <header class="main-header">
+    <a class="main-header__logo logo" href="index.html">
+      <img class="logo__image logo-image" src="img/logo-mobile.svg" height="38" width="33" alt="Надпись Кот Энерджи.">
+      <img class="logo__text logo-text" src="img/logo-text.svg" height="18" width="101" alt="Логотип Кот Энерджи.">
+    </a>
+    <nav class="main-nav">
+      <button class="main-nav__toggle" type="button">
+        <span class="visually-hidden">Открыть меню</span>
+      </button>
+      <ul class="main-nav__list">
+        <li class="main-nav__item">
+          <a href="index.html">Главная</a>
+        </li>
+        <li class="main-nav__item">
+          <a href="catalog.html">Каталог продукции</a>
+        </li>
+        <li class="main-nav__item main-nav__item--current">
+          <a>Подбор программы</a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <main class="main-form">
+    <div class="main-form__container">
+      <h1 class="main-form__title">Подбор программы</h1>
+      <p class="main-title title">Заполните анкету, и мы подберем программу питания для вашего кота</p>
+      <form class="main-form__selection-form" action="https://echo.htmlacademy.ru" method="get" name="selector">
+        <fieldset class="main-form__personality form form--primary">
```

Comment:
А здесь классы ` form`, `form--primary` не нужны.
```suggestion
        <fieldset class="main-form__personality">
```
Логичнее вообще такой класс:
```suggestion
        <fieldset class="form__fieldset">
```